prog/dftb+/lib_dftbplus/transportio.F90

```diff
@@ -53,7 +53,7 @@ subroutine writeShifts(fShifts, orb, shiftPerL)
       call error("Internal error in writeshift size(shiftPerL,2)")
     endif
 
-    open(newunit=fdHS, file=trim(fShifts), form="formatted")
+    open(newunit=fdHS, file=fShifts, form="formatted")
```

Comment:
What is the reason for dropping the trim (what happens if fShifts is padded with spaces)?

Reply:
Trim happens automatically in the `file=` construct as far as I remember.

Reply:
OK, I have found it in the Fortran 2003 standard draft:
 ```
The value of the FILE= specifier is the name of the file to be connected to the specified unit. Any 
trailing blanks are ignored.
```
So, yes, we can leave away `trim()` when passing character variables to the `file=` keyword.